pkg/acceptance/helpers/common_test.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMatchAllStringsInOrder(t *testing.T) {
+	testCases := map[string]struct {
```

Comment:
Please add tests for:
- different order of parts (to validate if order really matter)
- fancy case like "abba" with []string{"abb", "bba"} - this should not match given the current implementation, right? but from the method signature it could be expected to

Reply:
Done. Renamed the function.